lint/capability_fields_analyser.go

```diff
@@ -20,35 +20,69 @@ package lint
 
 import (
 	"github.com/onflow/cadence/runtime/ast"
+	"github.com/onflow/cadence/runtime/common"
 	"github.com/onflow/cadence/tools/analysis"
 )
 
-func DetectCapabilityType(typeToCheck ast.Type) bool {
+func DetectCapabilityType(typeToCheck ast.Type, structWithPubCapabilities map[string]struct{}) bool {
 	const capabilityTypeName = "Capability"
```

Comment:
prob worth making it global const with Capital name, wyt? )

Reply:
I don't think that it needs to be global.